### ISSUE TITLE
Is there anyway to exclude the js file from the bundle?

### ISSUE THREAD

**Issue by NullVoxPopuli** [FortAwesome/ember-fontawesome#52]
I'm looking at broccoli-concat-analyser, and see:

![image](https://user-images.githubusercontent.com/199018/41912385-931d79b4-791d-11e8-9e73-64357786a9ab.png)

it's in tiny text on the right. what's the js for? can I remove it somehow?

**Comment by robmadole** [comment]
Are you talking about the `fontawesome.js`? The 7.87 KB entry?

**Comment by NullVoxPopuli** [comment]
Yeah, that one :)

**Comment by NullVoxPopuli** [comment]
Yeah, ember-fontawesome https://gitlab.com/NullVoxPopuli/emberclear/blob/master/packages/frontend/package.json#L39

I'm both prerendering and not prerendering, but I was hoping that the fa icon helper could be replaced by i tags at build time, ya know? Just so there is no runtime evaluation / rendering of the helper (so, the Js then wouldn't be required :) )

**Comment by jrjohnson** [comment]
@NullVoxPopuli it sounds doable to have a configuration option to only allow build time rendering and never include the component or any of the backing JS from fontawesome. I'm not volunteering mind you :)

**Comment by burritoIand** [comment]
any movement on this?

**Comment by jrjohnson** [comment]
@burritoIand not yet. If you're interested in getting involved and helping jump into #e-fontawesome in discord and let's chat.
